tests/installation/welcome.pm

```diff
@@ -155,7 +155,7 @@ sub run {
 
     ensure_fullscreen;
 
-    if (is_sle_micro || (is_sle('15+') && get_var('UPGRADE'))) {
+    if (is_sle('15+') && get_var('UPGRADE')) {
```

Comment:
This change doesn't seem to be directly related to ssh installations?

Reply:
Yes and no. It was failing for SSH case. `inst-welcome` shall be used for both, regular and ssh installation.